posthog/api/organization_member.py

```diff
@@ -77,8 +71,9 @@ class OrganizationMemberViewSet(
 ):
     serializer_class = OrganizationMemberSerializer
     permission_classes = [IsAuthenticated, OrganizationMemberPermissions, OrganizationMemberObjectPermissions]
-    queryset = OrganizationMembership.objects.exclude(user__email__endswith="@posthogbot.user")
-    lookup_field = "user_id"
+    pagination_class = None
```

Comment:
I'm opposed to this change. IMO payloads for all listing endpoints should follow the same shape (below) for consistency, but keen on understanding your rationale behind this.

```json
{
    "count": 0,
    "next": null,
    "previous": null,
    "results": []
}
```

Reply:
The simple reason is that we currently don't support this pagination in the frontend, as the nature of this endpoint (organizations don't have that many members, and even if they do, this is loaded in very specific circumstances) makes it not worth it to implement frontend logic for handling `next`. But without this logic, if an organization _does_ have that many members, they will only see a limited number of them – because of pagination.
So the only thing this pagination results in is: the endpoint follow a somewhat standard shape BUT there's useless boilerplate in the response, which also is a potential bug cause if we don't expend extra effort on frontend logic for handling an edge case.
We could also set page size to something like 9999 and avoid the edge case… but that makes this even more hacky and the boilerplate is even more useless.

Reply:
I would just pass the limit / page size arg in the query string. I don't think it's hacky. Frontend can choose to handle pagination logic as it makes sense, in this case setting a large limit makes sense.

Reply:
Well, alright, brought pagination back, setting limit to 200 instead of the default 100 frontend-side.